sentry-samples/sentry-samples-android/.gitignore

```diff
@@ -1 +1,2 @@
 /build
+sentry-samples-android-keystore
```

Comment:
could use this one https://github.com/getsentry/sentry-java/blob/main/debug.keystore